packages/app/src/app/overmind/effects/api/apiFactory.ts

```diff
@@ -4,10 +4,18 @@ import { camelizeKeys, decamelizeKeys } from 'humps';
 
 const API_ROOT = '/api';
 
-// If the path starts with `/beta`, do not append
-// `/v1` to the api root url.
-const getBaseApi = (path: string) =>
-  path.startsWith('/beta') ? API_ROOT : `${API_ROOT}/v1`;
+/**
+ * If the path starts with `/beta`, do not append `/v1` to the api root
+ * url. Alternatively we have the useRoot param for when we just want to
+ * use the root path.
+ */
+const getBaseApi = (path: string, useRoot: boolean = false) => {
```

Comment:
Adding `useRoot` was the simplest change to make sure we were able to reuse most of the `api` and `apiFactory` logic, without having to write custom fetch methods or update the factory in a big way.